activerecord/test/models/author_encrypted.rb

```diff
@@ -2,6 +2,8 @@
 
 require "models/author"
 
+ActiveRecord::Encryption.config.add_to_filter_parameters = false
```

Comment:
It's interesting to turn this off when loading the model 🤔

Reply:
https://github.com/rails/rails/pull/55939#discussion_r2442597233

Yeah I don't like it either. Let's change the default instead of this hack.